doc/api/util.md

```diff
@@ -14,6 +14,60 @@ it:
 const util = require('node:util');
 ```
 
+## `util.addSafeAbortSignalAbortListener(signal, resource)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
```

Comment:
I was not sure about this... Should this be experimental?

Reply:
Absolutely